pydomcfg/utils.py

```diff
@@ -59,6 +56,8 @@ def _calc_rmax(depth: DataArray) -> float:
         depth_diff = depth.diff(dim)
         depth_rolling_sum = depth.rolling({dim: 2}).sum().dropna(dim)
         rmax = depth_diff / depth_rolling_sum
+        # dealing with nans at land points
```

Comment:
@jdha and @malmans2  I think we need something like this for managing nans (0/0) in the case of land points #40